### ISSUE TITLE
[ProgressView] Error support for progress views

### ISSUE THREAD

**Issue by yarneo** [material-components/material-components-ios#4308]
It seems like there isn't any current support for errors in a progress view. If the progress stops, times out, or errors, we should probably provide some kind of visual and voice notification letting the user know.

**Comment by romoore** [comment]
I might suggest that whatever view contains the progress view should handle errors. Since the progress view itself typically disappears (e.g., the Chrome progress view at the bottom of the header), the error message should probably be displayed elsewhere.

**Comment by ianegordon** [comment]
Agree with @romoore that error handling should be handled at a view controller lever.
Please re-open if you would like to discuss further.
